politeiawww/user/localdb/localdb.go

```diff
@@ -211,7 +214,8 @@ func (l *localdb) UserGetByPubKey(pubKey string) (*user.User, error) {
 // the public keys in the user's identity history.
 //
 // UsersGetByPubKey satisfies the Database interface.
-func (l *localdb) UsersGetByPubKey(pubKeys []string) (map[string]user.User, error) {
+func (l *localdb) UsersGetByPubKey(pubKeys []string) (map[string]user.User,
```

Comment:
Keep function declarations on a single line.  Same for PluginExec.

Reply:
Done.